website/src/hooks/useSearchAvailable.js

```diff
@@ -0,0 +1,4 @@
+const useSearchAvailable = () =>
+  !process.env.ALGOLIA_API_KEY || !process.env.ALGOLIA_INDEX ? false : true;
```

Comment:
environment doesn't change in run time, so I don't think there's a need to make this a hook eh :) can just export a constant called `isSearchAvailable` or something like that

Reply:
nice! thank you

Reply:
@hlminh2000 changed to exported constant from `utils`